.github/workflows/release.yml

```diff
@@ -0,0 +1,73 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  tag:
+    name: Tag new version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Bump version and push tag
+        run: "bash release"
+
+  release:
+    name: Create Release
+    needs: tag
```

Comment:
I think this will run with any tag, can it check that it only runs on `release-*`?

Reply:
Actually @snoe this workflow will run at every push on master which I think we should release a new tag when this happens don't you think?

When this workflow runs, it will run the `release` script tagging on github, and this `needs` tells to github action to only run this `release` step, after the `tag` step before.

Reply:
@ericdallo hrm, oh I missed the `run` above. I guess we can do this, personally I like more control over releases but let's try it and see how it works out.